keyboards/tiger910/keymaps/rev1/config.h

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
This should be removed

Reply:
This version has changed the number of RGB lights and needs to modify the configuration

Reply:
Specifically, this should not be in a keymap folder.  It should be in `/tiger910/rev1/` instead.

Reply:
> Specifically, this should not be in a keymap folder.  It should be in `/tiger910/rev1/` instead. 

I tried to do this, but I got compile errors

Reply:
That may be due to the naming and such.  If you post the errors, and push the code, we can take a look and help you fix those errors.

Reply:
OK.
I've made some changes.